### PR TITLE
Sort out imports and fix package build metadata, including list of dependencies

### PR DESCRIPTION
**Pull request by legaul-cisco**
- Sort out imports:
  - Organise into the following groups (following https://peps.python.org/pep-0008/#imports): stdlib, 3rd party, cafykit, relative
  - Remove unused imports
- Fix `setup.py` metadata:
  - Remove incorrect/irrelevant fields (e.g. no license)
  - List all dependencies (as per 3rd party imports group)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
 from setuptools import setup
 
 from cafy_pytest.__version__ import __version__
-#__version__ = "0.1.0"  
 
 setup(
     name="cafy_pytest",
     packages=['cafy_pytest'],
     package_data={'cafy_pytest': ['resources/*']},
-    author='Cafy',
```
Comment (shreyashah) [inline setup.py:16]: Why is this removed?

Comment (legaul-cisco) [reply setup.py:16]: The "author" field is normally someone's name, rather than a project name. I can add it back if you'd like.

**Review by nishantupadhyay123 — APPROVED** [review]

**Comment by legaul-cisco** [comment]
I think this just needs merging (which I don't seem to be able to do)
